pogom/search.py

```
@@ -329,6 +331,7 @@ def search_overseer_thread(args, new_location_queue, pause_bit, heartb,
     account_queue = Queue()
     threadStatus = {}
     key_scheduler = None
+    cur_plfe_version = None
```

1. Add current version as a clear, hardcoded constant in the scanner.
2. Add a `--no-version-check` config item. If the version check is not disabled (i.e. we want the version check), the map retrieves the current forced version and only starts scanning if it's the same version as the hardcoded value (on failure, don't start scanning either). Otherwise skip the check and just start scanning (but add a warning to logs that version check is being skipped). 
3. Since every new forced version requires a pgoapi update, updating the hardcoded value each time we update pgoapi is trivial (and most likely the best option).